mbtools/models.py

```diff
@@ -169,6 +171,25 @@ def __init__(self, parent, location):
         self.etree = html.fromstring(self.parent.text)
         # self.etree = html.fromstring(self.parent.text)
 
+    def replace_content_tag(self):
```

Comment:
I changed this to return `{"uuidvalue": "contentdata"}` if content was replaced and `None` if it was not. 

I am curious any reason why we want to return this `{"uuid": "uuidvalue", "content": "contentdata"}` instead of {"uuidvalue": "contentdata"}?  🤔

Reply:
This is somewhat subjective, but using the approach of `{"uuid": "uuidvalue", "content": "contentdata"}` has some benefits I think:

1. It's more obvious to readers of caller code like `result = html_elem.replace_content_tag()` that `result["uuid"]` is the content UUID versus the convention that it's the key in the returned dict
2. It may be a bit more extensible if we needed to return more data later (e.g. adding key "foobar" to the dict returned).
3. You can pass the result as named values to a function with `**result` later if desired.